plugins/README.md

```diff
@@ -36,6 +37,22 @@ Tips:
 -   To see a trace of logs and wasm ABI calls, add `--loglevel=TRACE`.
 -   Optionally specify plugin config data using the `--config=<path>` flag.
 
+# Building Plugins
+To build your plugin using bazel, run:
+
+```
+bazelisk build <BUILD Target>
+```
+For example, to build the sample add_header plugin in C++, from the `plugins/` subdirectory of this repository, run:
+
+```
+bazelisk build //samples/add_header:plugin_cpp.wasm
+```
+If attempts to build fail due to errors in dependencies, you may need to add ```--config=clang``` or ```--config=gcc``` to the build command.
```

Comment:
Use single \` characters here, not triple \`, which is only needed for multiline code blocks.